### DISCUSSION TITLE
Using multiple apps for pin mode?

### DISCUSSION THREAD

**Discussion by aashiqsworld** [rxhanson/RectanglePro-Community#82] [Ideas]
Hey! I'm a long time user of Rectangle and just found out Rectangle pro exists, I immediately upgraded. I was hoping there would be more flexibility for todo/pin mode but didn't find anything different. Are there any plans to allow more than one application to be pinned, one on the top half of the side of the screen and the other on the bottom half? I'm sure other people with big monitors would appreciate it as well 😊 

Thanks in advance! And thanks for developing Rectangle, it has sped up my workflow considerably.

**Comment by rxhanson** [comment]
There are plans for this, and that's part of why I changed the name to Pin mode in Pro. Pro does already have some additional preferences, like the ability to select the left side. (Thanks for making the jump to Pro!)

**Comment by aashiqsworld** [comment]
Awesome, I'll look forward to this feature 😁

**Comment by andyngo** [comment]
My apologies for necro-ing this discussion but figured it'd be better to continue the discussion here than to open a new one.

I've been looking for a way to use 2 pinned similar to the attached image. Is there a way to do this with the app at the moment? Thanks!

![image](https://github.com/rxhanson/RectanglePro-Community/assets/278293/433e9096-78a3-4f89-a98c-5de98cca0052)

**Reply by rxhanson** [reply]
Thanks for asking, this one never quite bubbled up to the top of my queue. I'll add your +1 to it, though.

**Reply by andyngo** [reply]
Thank you! I realised I didn't need my to-do list to take up the whole vertical screen real-estate. Would be nice to be able to pin something else (ie: Terminal / YouTube PWA) down there.
